code/modules/projectiles/guns/revolvers.dm

```diff
@@ -267,6 +267,51 @@
 	scatter = 8 // Only affects buckshot considering marksman has -15 scatter.
 	damage_falloff_mult = 1.2
 
+//-------------------------------------------------------
+// The R-76 Magnum. Fires a big round, equal to a slug. Has a windup.
+
+/obj/item/weapon/gun/revolver/standard_magnum
+	name = "\improper R-76 KC magnum"
+	desc = "The R-76 magnum is an absolute beast of a handgun used by the TGMC, rumors say it was created as a money laundering scheme by some general due to the sheer inpracticality of this firearm. Hits hard, recommended to be used with its stock attachment. Chambered in 12.7mm."
```

Comment:
"The R-76 magnum is an absolute beast of a handgun used by the TGMC. Rumors say it was created as a money laundering scheme by some general due to the sheer inpracticality of this firearm. Hits hard: recommended to be used with its stock attachment. Chambered in 12.7mm."

There's more to mid-sentence punctuation than commas.